superbit_lensing/medsmaker/scripts/process_mocks.py

```diff
@@ -29,10 +29,10 @@
                     help='Set to keep coadd cutout in MEDS file')
 parser.add_argument('--clobber', action='store_true', default=False,
                     help='Set to overwrite files')
-parser.add_argument('--source_select', action='store_true', default=False,
+parser.add_argument('--source_select', action='store', default=False, choices=['true', 'True','false','False'],
                     help='Set to select sources during MEDS creation')
-parser.add_argument('--select_stars', action='store_true', default=False,
-                    help='Set to remove stars during source selection')
+parser.add_argument('--select_truth_stars', action='store', default=False,
```

Comment:
I prefer `store_true` as well, but it's not so easy to pass something from the configuration file as just a flag. It's done here for the verbose flag: https://github.com/superbit-collaboration/superbit-metacal/blob/4e7e40a6d386b47b9d381f7ac71af73ef3460e11/superbit_lensing/pipe.py#L72 

I was trying to make as few changes as possible to `pipe.py` but perhaps we could add a similar statement in the module here `class MedsmakerModule(SuperBITModule):`

Reply:
as discussed on slack, pipe.py is being updated to handle this:
https://github.com/superbit-collaboration/superbit-metacal/blob/shear-refactor/superbit_lensing/pipe.py#L81

Reply:
This still needs to be set to `action=store_true` (and also delete the unnecessary empty line)

Reply:
also I promise I'm not trying to be annoying, but it would be nice if we would follow a single syntax style to make it easier to read. So spaces after commas, etc.

Reply:
(1) Done and (2) you're right. Is there a style guide you would like to use?

Reply:
Ideally we'd use a code linter to do this automatically for us, but I haven't been fancy enough to set that up yet (and would require buy-in from both of us). I'm not a stickler, but a few things like spaces after commas in most instances, no extra whitespace lying around, max 80-100 characters per line, etc. is usually a good call for readability